src/utils.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+/* eslint-env browser, webextensions */
+
+const multiaddr = require('multiaddr')
+
+function isMultiaddress (addr) {
+  if (addr === null || addr === undefined || typeof addr === 'undefined') {
+    return false
+  }
+
```

Comment:
sneaked in during refactoring, can be removed, try/catch below is enough

Reply:
This is on `isMultiaddr` and not on `isUrl`. Our `multiaddr` does not throw with undefined nor null. I'll leave this check, but the one on `isUrl` was removed.